helm/kubernetes-cluster-prep/templates/tests/_helm_test.bats.txt

```diff
@@ -0,0 +1,33 @@
+# Define a Helm named template so that this BATS script can be included
+# in a Helm test ConfigMap. This can be left commented out; Helm will
+# still be able to find this template, and this file can be kept as
+# valid BATS syntax.
+#
+{{- define "kube-cluster-prep.helm-test.bats" }}
+#!/usr/bin/env bats
+
+# Run out of same directory in which this script resides
+cd "/tests"
+source "./utils.sh"
+
+source "/bats/bats-support/load.bash"
+source "/bats/bats-assert/load.bash"
+source "/bats/bats-file/load.bash"
+
+# Baseline BATS test result color
+text_color "$MAGENTA"
+
+@test "Conjur Appliance URL is a reachable address" {
+  status=0
```

Comment:
We can leave out the initialization of the `status` variable. The `run` command always declares/initializes that for us automatically.

I should have added the documentation for this earlier:
    https://bats-core.readthedocs.io/en/latest/writing-tests.html

Based on that documentation ^^^ (and the example it provides), the `run` command gives us these three variables automagically:
- $status
- $output
- $lines